native_client/beam_search.h

```diff
@@ -53,7 +53,7 @@ class KenLMBeamScorer : public tensorflow::ctc::BaseBeamScorer<KenLMBeamState> {
   // ExpandState is called when expanding a beam to one of its children.
   // Called at most once per child beam. In the simplest case, no state
   // expansion is done.
-  void ExpandState(const KenLMBeamState& from_state, int from_label,
+  void ExpandState(const KenLMBeamState& from_state, int /*from_label*/,
```

Comment:
Why is this commented and not the rest?

Reply:
Because that's the only parameter that isn't used. This was done to silence a warning.